services/core-web/common/constants/API.js

```diff
@@ -20,7 +20,10 @@ export const MINE_NAME_LIST = (params = {}) => `/mines/search?${queryString.stri
 export const MINE_STATUS = "/mines/status";
 export const MINE_REGION = "/mines/region";
 export const MINE_COMPLIANCE_SUMMARY = (mine_guid) => `/mines/${mine_guid}/compliance/summary`;
-export const MINE_TSF = (mine_guid) => `/mines/${mine_guid}/tailings`;
+export const MINE_TSF = (mine_guid, mine_tailings_storage_facility_guid) =>
```

Comment:
Can you please break this up into two separate constants? It would be more similar to how we've done others.

E.g.: MINE_TSF and MINE_TSFS

Reply:
Decoupled the variables.